test_cases/assert_violations/runtime_create_user_input.yaml

```diff
@@ -1,8 +1,9 @@
-description: "Assert violation after call to contract created with user input during runtime"
+description: Assert violation after call to contract created with user input during runtime
 issues:
-- id: "SWC-110"
+- id: SWC-110
   count: 1
   locations:
-  - bytecode_offsets: [296]
-    line_numbers: [11]
-
+  - bytecode_offsets:
+      '0x8b3686fb2c2aa036f95720baaa2cc9c033c4b0a9611870d8db0a84aca15d9a27': [296]
```

Comment:
I think this should be the following instead:
336:1:0 (0x1410f656e6c549fa79b9b13b559ef15eb4149c80d4f22a314642993add450e19)

Reply:
@s0b0lev @thec00n BTW, does Edelweiss run on each contract separately?

Reply:
yes